### DISCUSSION TITLE
Cron example and web deploy console silent errors

### DISCUSSION THREAD

**Discussion by kluesing** [vercel/community#3063] [Bug Report]
### Page to Investigate

https://vercel.com/new/clone

### Steps to Reproduce

Using https://vercel.com/templates/next.js/vercel-cron on the Hobby plan there's a silent failure in the web based deploy wizard. 

1. New user to Vercel
2. Navigate to https://vercel.com/templates/next.js/vercel-cron and click "Deploy"
3. Link GitHub account on web deploy helper (https://vercel.com/new/clone?repository-url=) 
4. Configure KV storage
5. Web deploy helper hangs on "Deploy" forever

It looks like the Cron project has cron jobs at 1m/1h etc that are below the account limits for the Hobby plan. At the CLI, these error out with helpful error messages. But, the web console doesn't pass these errors through, so a first time user ends up stuck with a hung page.

(It's easy to fix the cron template once you move to the CLI - just delete the offending cron frequencies - the bug report is for the web console silently eating the errors and hanging on first user experience)

**Comment by amyegan** [comment]
Thanks for reporting this! I'll open a ticket so we can get a proper warning on the project creation page.
